### PR TITLE
[AG-15325] Maintenance(ci): integrate slack gha into docs build workflow

### PR DESCRIPTION
**Pull request by sdwvit**
- Add CTRF-to-Slack blocks conversion script for better test reporting
- Update JIRA integration action to reference new file location
- Move create-jira-ticket.mjs to jira-integration directory for organized structure
- Enable Slack notification placeholder with false condition for future activation

Using a fork of github-test-reporter as it was missing an essential feature. Created a PR upstream to address that https://github.com/ctrf-io/github-test-reporter/pull/160.

### CODE REVIEW

File: .github/workflows/doc-tests.yml [hunk 1]
```diff
@@ -73,6 +79,7 @@ jobs:
     download-examples:
       name: Download all-examples.json
       runs-on: ubuntu-latest
+      if: ${{ github.event.inputs.skip_vanilla != 'true' || github.event.inputs.skip_typescript != 'true' || github.event.inputs.skip_reactFunctional != 'true' || github.event.inputs.skip_reactFunctionalTs != 'true' || github.event.inputs.skip_angular != 'true' || github.event.inputs.skip_vue3 != 'true'}}
```
Comment (StephenCooper) [inline .github/workflows/doc-tests.yml]: Is this really required? Guessing to cover an odd edge case where none of the tests are run, but even then is there harm in the download running? 

This seems like a lot of extra noise for that case.

Comment (sdwvit) [reply .github/workflows/doc-tests.yml]: yea, lets remove it 
```suggestion
```

File: .github/workflows/doc-tests.yml [hunk 2]
```diff
@@ -204,10 +211,63 @@ jobs:
             path: combined-reports
 
         - name: Publish Combined Test Report
-          uses: ctrf-io/github-test-reporter@v1
+          id: ctrf
+          uses: sdwvit/github-test-reporter@v1.0.15report6
```
Comment (StephenCooper) [inline .github/workflows/doc-tests.yml:214]: Why does this one have your id `sdwvit` in it?

Comment (sdwvit) [reply .github/workflows/doc-tests.yml:214]: Fair question. 
This is a fork, I have opened a pull request against base to add missing output. 

Once this stabilizes, either proposed change gets merged or I transfer the fork to AgGrid ownership, then I update this line.

Comment (sdwvit) [reply .github/workflows/doc-tests.yml:214]: here is a proposed change https://github.com/ctrf-io/github-test-reporter/pull/156

**Review by StephenCooper — APPROVED** [review]

**Review by StephenCooper — APPROVED** [review]
